js/packages/sdk-utils/src/lib/logger.js

```diff
@@ -0,0 +1,34 @@
+import { createLogger, transports, format, config } from 'winston'
+import chalk from 'chalk'
```

Comment:
@pinanks why we are not using logger package, or sdk logger

Reply:
@sushobhit-lt there's no need to create a separate logger package, it will be a part of sdk-utils itself